blockbuster/tests/bubblegum_parser_test.rs

```diff
@@ -96,7 +96,10 @@ fn test_basic_success_parsing() {
         slot: 0,
     };
     let result = subject.handle_instruction(&bundle);
-    assert!(result.is_ok());
+    if let Err(err) = result {
```

Comment:
I just wanted it to print out the error in the case of it failing the test.  I was working on the test and was just getting "panic at result is not OK", and wanted to read the message.  Since it was a test I did not spend too much time crafting  the best new version.

Reply:
Yeah, I was just wondering because IIRC simply using `unwrap` or `expect` will by default print the error as well if it implements `Debug`, but that's just trivia rn.

Reply:
You are right unwrap will do it.  Removed the extraneous print.